GearWorksXO-2017/src/org/usfirst/frc/team2212/subsystems/Lift.java

```diff
@@ -0,0 +1,61 @@
+package org.usfirst.frc.team2212.subsystems;
+
+import java.util.function.Supplier;
+
+import com.spikes2212.dashboard.ConstantHandler;
+import com.spikes2212.genericsubsystems.LimitedSubsystem;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.PIDSource;
+import edu.wpi.first.wpilibj.SpeedController;
+
+
+public class Lift extends LimitedSubsystem {
+
+    public SpeedController liftMotor;
+    public DigitalInput upLimit;
```

Comment:
you don't need to call it liftMotor. it is obvious that this is the motor of the lift. same goes to the encoder